server/client.go

```diff
@@ -132,7 +179,6 @@ func (c *client) initClient() {
 	c.cid = atomic.AddUint64(&s.gcid, 1)
 	c.bw = bufio.NewWriterSize(c.nc, startBufSize)
 	c.subs = make(map[string]*subscription)
-	c.debug = (atomic.LoadInt32(&debug) != 0)
```

Comment:
Wasn't use anywhere. I can put back if you want. I left trace (and could put back debug) as booleans not included in flags because `c.trace` is in the fast path (check if message needs to be traced, etc..)